demo/demos/jsTransition/index.tsx

```diff
@@ -1,12 +1,37 @@
 import { defineComponent } from 'vue';
 
+import { Side } from '@floating-ui/vue';
 import { defineDemo, html } from '../../helpers';
-import { Wowerlay } from '../../../src/lib';
+import { Wowerlay, type WowerlayTransitionFn } from '../../../src/lib';
 import useDemoState from '../../helpers/useDemoState';
 
 const Component = defineComponent({
-  name: 'CustomTransition',
-  setup: () => useDemoState(),
+  name: 'JsTransition',
+  setup: () => {
+    const handleTransition: WowerlayTransitionFn = (type, element, done) => {
+      const placement = element.getAttribute('data-popover-placement')!.split('-').at(0) as Side;
+
+      const [from, to] = [
+        {
+          transform: `translateY(${placement === 'top' ? '10px' : '-10px'})`,
+          opacity: 0,
+        },
+        {
+          transform: `translateY(0px)`,
+          opacity: 1,
+        },
+      ];
```

Comment:
I think an array definition is not needed here, we can just define the keyframes in their own variables:
```js
	const from = {
          transform: `translateY(${placement === 'top' ? '10px' : '-10px'})`,
          opacity: 0,
        };
```